CHANGELOG.md

```diff
@@ -15,6 +15,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 
+## [0.2.0] - 2021-8-31
+
```

Comment:
You need to add a link target here. This is pretty cool, as it lets you click on the version number to get a diff from the previous release.

```suggestion
[0.2.0]: https://github.com/opendp/opendp/compare/v0.1.0...v0.2.0

```